ux.symfony.com/src/Entity/LiveMemory/Game.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Entity\LiveMemory;
+
+use Symfony\Component\Uid\Uuid;
+use Symfony\Component\Uid\UuidV7;
+
+/**
+ * @demo LiveMemory
+ *
+ * @author Simon André <smn.andre@gmail.com>
+ */
+class Game
```

Comment:
WDYT about also putting this in a `src/LiveMemory` directory (and moving the `src/Service/LiveMemory` services there too)? Everything else is nicely organized together. And while I know that "entity" is meant to be a generic term, in our world, it's understood as "Doctrine" entity. So keeping this elsewhere with the game stuff looks good to me.

Reply:
Yes of course... at first the next move was to save those and display highscores and such... but no need for now and i agree isolated code is easier to maintain.



**_LATER_**, pushing this idea just a bit..... wdyt using a dedicated namespace for _**all**_ code related to this demo ? 

I do thing with a few lines of code, we could use a bundle-like structure that'd make really easy to create/maintain/tests demos, and facilitate the making of new ones

```
├── assets
├── config
├── demo
│   └── live-memory
│       ├── assets
│       ├── src
│       │   ├── Controller
│       │   └── Twig
│       └── templates
│           └── components
├── src
│   ├── Controller
│   ├── Entity
│   ├── EventListener
│   └── Twig
└── templates

```

Reply:
Hmm, perhaps. What I want to avoid is user-facing "oddness". For example, in most demos, we should the files behind the demo - e.g. https://ux.symfony.com/demos/live-component/auto-validating-form#registrationform-php

We wouldn't want that to show `demo/auto-validation/src/Twig/RegistrationForm.php`. However, that's also something that could be hidden pretty easily.

So I'm 90% +1 for this idea for `live-memory`. For the other demos, I think we need to try it and see how it feels.

Reply:
> We wouldn't want that to show `demo/auto-validation/src/Twig/RegistrationForm.php`. However, that's also something that could be hidden pretty easily.

Of course. The question remains concerning PHP namespaces.. Do we want to _show_ `App/Twig/Component` or do we accept something like `App/Demo/LiveMemory/Twig/Component` ?

Reply:
We'll reopen this real soon, closing now (regrouped all class but components in a folder)